cli/tests/integration/js_unit_tests.rs

```diff
@@ -15,8 +19,106 @@ fn js_unit_tests_lint() {
   assert!(status.success());
 }
 
-#[test]
-fn js_unit_tests() {
+util::unit_test_factory!(
+  js_unit_test,
+  "tests/unit",
+  "*.ts",
+  [
+    abort_controller_test,
+    blob_test,
+    body_test,
+    broadcast_channel_test,
```

Comment:
Could this be changed to automatically read the directory for all files ending with `_test.ts`? I feel like it will be a footgun adding a new test file and wondering why it's not running

Reply:
It does already -- if the list doesn't match, there's a virtual unit test that fails. Basically you'll get a test failure that tells you what to update the list to.

Reply:
See https://github.com/denoland/deno/pull/19199/files#diff-c58588fbe47fc79480797c3896bec5dd1112c5d416242c96f6881e433083b119R27

Reply:
I updated the glob to use `_test.ts` and renamed a test that was getting skipped previously.